spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  user_id = 19
+  post_id = 19
+  describe 'GET /posts' do
+    it 'returns a successful response' do
+      get "/users/#{user_id}/posts"
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the index template' do
+      get "/users/#{user_id}/posts"
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get "/users/#{user_id}/posts"
+      expect(response.body).to include('Number of Posts')
+    end
+  end
+
+  describe 'GET /posts/:id' do
+    it 'returns a successful response' do
+      get "/users/#{user_id}/posts/#{post_id}"
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the show template' do
+      get "/users/#{user_id}/posts/#{post_id}"
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get "/users/#{user_id}/posts/#{post_id}"
+      expect(response.body).to include('Number of Post')
+    end
+  end
+end
```

Comment:
- [ ] I wanted to bring to your attention that there's an ongoing issue causing one of your tests to fail. The error message `Couldn't find User with 'id' = 19` indicates that the test is expecting a user with ID 19, but it seems that the necessary data might not be set up correctly in your test environment. It would be fantastic if you could investigate this situation and make the necessary adjustments to the test data or configuration so all tests can pass successfully.

 - ### ERROR

     ![image](https://github.com/ibugithub/R-Blog/assets/87197412/fb8d7d79-f4f8-4264-9955-399b369fd160)